tests/uncached.mjs

```diff
@@ -0,0 +1,112 @@
+import * as utils from '../utils.mjs';
+
+const tests = [];
+
+function defaultResponseHeaders (validatorType, validatorValue, additionalHeaders) {
+  return [
+    ['Cache-Control', `max-age=3600`],
+    ['Date', 0],
+    [validatorType, validatorValue]
+  ].concat(additionalHeaders);
+}
+
+function checkCached ({name, id, configuredHeaders, expectedHeaders, unexpectedHeaders=[]}) {
+  const etag = utils.httpContent(`${id}-etag-1`);
+  const etag1 = `"${etag}"`;
+  const lm1 = utils.httpDate(Date.now(), -24 * 60 * 60);
+  tests.push({
+    name: `${name} with a Last-Modified validator`,
+    id: `uncached-lm-${id}`,
+    requests: [
```

Comment:
This isn't specified behaviour yet, so I think this needs to be `kind: 'check'`.

Reply:
Well, except for the `no-cache` tests...

Reply:
Sounds good.